packages/shared.test/src/handlers/loadObjectsEndpoints.ts

```diff
@@ -493,7 +493,8 @@ export const loadObjectsEndpoints: Array<RequestHandler> = [
       const queryResponse = queryResponses[JSON.stringify(parsedBody)];
       if (
         req.params.ontologyApiName === defaultOntology.apiName
-        && queryResponse
+        || req.params.ontologyApiName === defaultOntology.rid
+          && queryResponse
```

Comment:
Also, the query endpoints were in this file for some reason, I just modified here for now, but I'm inclined to move them out into their own file